c++/src/kj/exception.c++

```diff
@@ -254,7 +254,7 @@ ArrayPtr<void* const> getStackTrace(ArrayPtr<void*> space, uint ignoreCount) {
 #endif
 }
 
-#if __GNUC__ && !_WIN32
+#if (__GNUC__ && !_WIN32) || __clang__
```

Comment:
Weak symbols are supported by Clang on Windows? But not by GCC (MinGW)? That seems surprising, I didn't think Windows had weak symbols, but if it does, you'd think  both compilers would support it.